test/e2e/cert_rotation_test.go

```diff
@@ -42,11 +40,10 @@ func certRotation() {
 
 	By("Get the current number of time containers were restarted")
 	containerNumRestart := make(map[string]int32)
-	containerNumRestart["ironic-api"] = 0
-	containerNumRestart["ironic-conductor"] = 0
+	containerNumRestart["ironic-httpd"] = 0
```

Comment:
should not we add ironic container also here ?

Reply:
ironic container is not needed as ironic-httpd is taking care of the TLS

Reply:
As ironic-httpd is taking care of TLS, we don't need ironic container